schema/partner_show.js

```diff
@@ -64,6 +65,11 @@ const PartnerShowType = new GraphQLObjectType({
     press_release: markdown(),
     start_at: date,
     end_at: date,
+    exhibition_period: {
```

Comment:
hah, just found this discussion - yeah, I think realistically exhibition period is a much better name.